src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2310,6 +2310,26 @@ void processContextElement
                     getEntitiesCollectionName(tenant).c_str(),
                     query.toString().c_str()));
 
+    std::map<std::string, std::string>::iterator it = uriParams.find("op");
+
+    // if is an append and uri has not possibility of options
+    if (strcasecmp(action.c_str(), "append") == 0 && it == uriParams.end())
```

Comment:
The "semantics" used by the mongoBackend module are APPEND and APPEND_ONLY (the op=append is only used in the serviceRoutine layer which invokese mongoBackend, i.e. if the cliente resquest uses op=append, then serviceRoutings translate it to APPEND_ONLY before invoking mongoBackend).

Thus, probably this statement could be simplified to:

```
if (strcasecmp(action.c_str(), "append_strict"))
```

and the `it` variable is not actually needed.